test/test_helper.rb

```diff
@@ -25,6 +25,67 @@
 
 require 'sucker_punch/testing/inline'
 
+# Mock up vault client
+class VaultClient
```

Comment:
named it this way on purpose, otherwise there needs to be logic in the storage engine to know which one to instantiate.